MyCoolApi/MyCoolApi/DAL/CleatRepository.cs

```diff
@@ -0,0 +1,67 @@
+﻿using MyCoolApi.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using MyCoolApi.Models;
+
+namespace MyCoolApi.DAL
+{
+    public class CleatRepository : ICleatRepository
+    {
+        //not sure this will work
+        public Cleat AddNewCleat(Cleat newCleat)
+        {
+            if (newCleat == null) return null;
+            else
+            {
+                var addedCleat = new Cleat
+                {
+                    Name = newCleat.Name,
+                    Price = newCleat.Price,
+                    PartNumber = newCleat.PartNumber,
+                    Color = newCleat.Color,
+                    Sport = newCleat.Sport,
+                    Brand = newCleat.Brand
+                };
+                return addedCleat;
+            }
+        }
+
+        public List<Cleat> GetAllCleats() //don't think this is enough, not sure how to get all cleats
+        {
+            var allCleats = new List<Cleat>();
```

Comment:
How do I populate it if I don't have a database? Do I make up a fake list of cleats that I loop through to make. List?